python/cell_reader.py

```diff
@@ -195,6 +196,69 @@ def get_factors(metadata):
     return factors
 
 
+def get_cell_sets(clusters, lookup):
+    '''
```

Comment:
not sure if the `namedtuple` import can be done within the doctest, but does this work?

```suggestion
    '''
    >>> from collections import namedtuple
```